src/components/Events.vue

```diff
@@ -47,31 +47,43 @@ export default {
       },
       {
         id: 1,
         date: getDateParts(new Date('2022-02-02')),
-        title: 'Workshop: Intro-level picoMini CTF - UIO and online - 17.00-20.00',
+        title: 'Workshop: Intro-level picoMini CTF - Caml at UIO and online - 17.00-20.00',
         description: 'From picoctf.org: "Recommended for players interested in experiencing a picoCTF competition who may feel intimidated by the content’s difficulty and competitiveness of our annual contest. This CTF consists of introductory level challenges requiring no previous experience to play. Some programming knowledge is helpful as challenges progress. Designed for individual play." The picoMini CTF for noobies runs from Jan 10 - Feb 4! Oslo CTF host an event to meet up and hack together at the very end of the CTF. This event is for everyone who wants to practice CTF skills or learn some new skills. No previous experience is required, only a will to do some thorough googling! There is something for everyone due to the categories and levels. If it is a little too hard, we know other challenges to recommend. When logged in to play.picoctf.org, you may find the CTF here: https://play.picoctf.org/events/69'
       },
       {
         id: 2,
-        date: getDateParts(new Date('2022-03-23')),
-        title: 'Workshop: Introduction to Hack The Box (HTB) - UIO and online - 17.00-20.00',
-        description: 'Beskrivelse kommer. Target audience: Everyone interested in checking out Hack The Box. This is especially nice for those who would like to understand more of pentesting.'
+        date: getDateParts(new Date('2022-02-23')),
+        title: 'Workshop: CTI -> Intelligence Requirements -> Where Technical and Tactical Analysis meets the Business Needs - Caml at UIO and online - 17.00-20.00',
+        description: 'First we will go through what cyber threat intelligence is in context of the intelligence process. Then we will convert a small portion of that knowledge into practice and work together to identify potential stakeholders and their needs, prioritize and track their requirements, and translate those requirements into collection efforts and analysis.'
       },
       {
         id: 3,
+        date: getDateParts(new Date('2022-03-09')),
+        title: 'Workshop: Web hacking 101 - Caml at UIO and online - 17.00-20.00',
```

Comment:
Jeg har gjort en feil her ser jeg, foredragsholder sa det kan bli litt vanskelig med stream. Også var den første tittelen og beskrivelser bare noe jeg skrev etter samtale med han, mens han nå har oppdatert teksten sin, så det er et par endringer til her.

```suggestion
        title: 'Workshop: Introduksjon til Web Hacking - Caml at UiO (not online) - 17.00-20.00',
```